### ISSUE TITLE
Beautify php format bug

### ISSUE THREAD

**Issue by hswhsw610** [ajaxorg/ace#4670]
![image](https://user-images.githubusercontent.com/48399282/161690675-694ca953-2409-4398-9f7a-e27116599929.png)
if i format this
![image](https://user-images.githubusercontent.com/48399282/161690706-ae6bf6ee-e9ec-4de3-afef-f3ebe61fc13e.png)
it changes like this.

**Comment by andrewnester** [comment]
Could you please provide a bit more details on what are doing in your example, what's current and what's expected behaviour?

**Comment by hswhsw610** [comment]
Umm... It's simple.
I expected it to be formatted as above one but instead, it breakdowns to below

**Comment by andrewnester** [comment]
Okay, the issue is indeed easy to reproduce and comes down to this code in beautify extension
https://github.com/ajaxorg/ace/blob/master/lib/ace/ext/beautify.js#L272-L273

**Comment by andrewnester** [comment]
I guess the option we have is to expose formatting options to be configured which will allow to change this behaviour but by default it will stay the same, so no breaking changes to formatter

I’ll work on PR for this

**Comment by andrewnester** [comment]
The change has been released in 1.50.0 version, now you can configure formatting behaviour by setting
```
beautify.formatOptions.lineBreaksAfterCommasInCurlyBlock = false
```
